<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgConnexion</class>
 <widget class="QDialog" name="DlgConnexion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection AppliBDD</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/logo/icone/logo.png</normaloff>:/logo/icone/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelUser">
     <property name="text">
      <string>User :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditUser"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>Password :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="text">
      <string>azerty</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Server :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditServer">
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="butCancel">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="butConnexion">
     <property name="text">
      <string>&amp;Connection</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditUser</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>lineEditServer</tabstop>
  <tabstop>butConnexion</tabstop>
  <tabstop>butCancel</tabstop>
 </tabstops>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
